lib/project_types/theme/commands/package.rb

```diff
@@ -53,6 +56,20 @@ def theme_name(path)
 
         [theme_name, theme_info["theme_version"]].compact.join("-")
       end
+
+      def command
+        return @command unless @command.nil?
+
+        @command = if @ctx.which(ZIP) == true
+          ZIP
+        elsif @ctx.which(SEVEN_ZIP) == true
+          SEVEN_ZIP
+        end
+      end
```

Comment:
so originally this method looked like this:
```
def command
  @command ||= (if @ctx.which(ZIP) == true
    ZIP
  elsif @ctx.which(SEVEN_ZIP) == true
    SEVEN_ZIP
  end)
end
```
however rubocop kept removing the parentheses. these were necessary because `||=` wasn't working as expected without the parentheses and evaluated the if statement even when `@command` was set. 

so! let me know if you hate this and want me to try something else